tests/unit/test-wcs-cart-functions.php

```diff
@@ -9,7 +9,7 @@ class WCS_Cart_Functions_Test extends WP_UnitTestCase {
 	*/
 	public function provider_mock_cart() {
 		return array(
-			array( $this->getMockBuilder( 'WC_Cart' )->setMethods( array( 'get_cart', 'get_item_data' ) )->getMock() ),
+			array( $this->getMockBuilder( 'WC_Cart' )->setMethods( array( 'get_cart', 'get_item_data' ) )->disableOriginalConstructor()->getMock() ),
```

Comment:
Just noting here. If I didn't disable the constructors, all these mock cart instances would have actions hooked onto `woocommerce_add_to_cart` and so if a product was added to any cart, all these instances would also recalculate their totals.